### ISSUE TITLE
Broken in 10.10 Yosemite?

### ISSUE THREAD

**Issue by skyepn** [planbnet/QuickNFO#11]
Is QuickNFO.qlgenerator working for anyone with OS X 10.10 Yosemite?  
Doesn't seem to be working for me.....

**Comment by gitguys** [comment]
LEMONed, will that work in Mavericks 10.9.5 too?  Thank you.

**Comment by LEMONed** [comment]
@gitguys Yes, it works perfectly with 10.9.5.

**Comment by Seacidal** [comment]
The link to LEMONed's version is dead and I'm very ignorant when it comes to Xcode. Any chance of a re-up?  Thanks!

**Comment by babalooza** [comment]
+1
Like Seacidal I'd appreciate a download option for a version compiled with the latest Xcode.
Or instructions how to do it on my own.
Thanks!

**Comment by Seacidal** [comment]
Thanks for the effort. Unfortunately, even with babalooza's update and Terminal commands, I'm still just getting a blank, black screen.

**Comment by lajosdavid** [comment]
It still works on El Capitan.
Download the compiled version here: [QuickNFO.qlgenerator.zip](https://github.com/planbnet/QuickNFO/files/172172/QuickNFO.qlgenerator.zip)
(Built with Xcode 7.2.1)
Install: Simply unzip and copy the _QuickNFO.qlgenerator_ to _Library/QuickLook_.

**Comment by vigo** [comment]
macOS Sierra :(

**Comment by LEMONed** [comment]
@vigo works fine on Sierra. Make sure it's placed under `/Library/QuickLook` instead on `~/Library/Quicklook`.

**Comment by vigo** [comment]
omg! @LEMONed thanx a lot man! shit! maybe some of other "not working" plugins work too? I usually put my stuff under `~/Library/Quicklook`. Again thanx toooo much!

**Comment by gingerbeardman** [comment]
Thanks for the new download and placement tip.

**Comment by allixsenos** [comment]
confirmed works in `/Library/Quicklook` on 10.12.2

**Comment by SainteCelery** [comment]
Is there a reason it can no longer be used on a per user install basis? Was it re-coded to specifically prevent single user installations? Or did Sierra screw something up with the user quicklook library? I like to keep the os library as clean of detritus from user installations as possible.

**Comment by nrlquaker** [comment]
Have someone tried it on macOS Sierra?
Its not working for me. I've rebuilt it with Xcode 9.0 - still not working.
Not from `~/Library/QuickLook`, not from `/Library/QuickLook`

**Comment by gingerbeardman** [comment]
@nrlquaker have you tried the download above?

**Comment by nrlquaker** [comment]
@gingerbeardman Yes, I've tried [this one](https://github.com/planbnet/QuickNFO/files/172172/QuickNFO.qlgenerator.zip).
Not working for me.

**Comment by gingerbeardman** [comment]
I'm not on 10.12.6, I'll have to wait until my wife's laptop is free.

Though I'm on 10.13 Beta (17A362a) and it works for me?

without QuickNFO.qlgenerator

<img width="918" alt="screen shot 2017-09-23 at 21 45 43" src="https://user-images.githubusercontent.com/49612/30776976-a4bca738-a0a8-11e7-8478-6dfe8f8b1b05.png">

with /Library/QuickLook/QuickNFO.qlgenerator

<img width="608" alt="screen shot 2017-09-23 at 21 46 10" src="https://user-images.githubusercontent.com/49612/30776978-ac7e47b0-a0a8-11e7-86e0-caf6ea8cc675.png">

**Comment by gingerbeardman** [comment]
My wife is on 10.12.5 (that's the best I can do, for now, I'm sure you understand)

Works there also:

![screen shot 2017-09-23 at 21 57 19](https://user-images.githubusercontent.com/49612/30777181-10fd9c14-a0ad-11e7-8b90-d9d817537bf5.png)

**Comment by nrlquaker** [comment]
Working fine at 10.13.

**Comment by LEMONed** [comment]
If you find it broken in 10.14.1, here's one rebuilt with Xcode 10.1, tested and working:

https://www.dropbox.com/s/9fmy5lqnh2kghr9/QuickNFO_Nov2018.zip?dl=0

and REMEMBER:
❌`~/Library/Quicklook/`
✅`/Library/Quicklook/`

**Comment by tessus** [comment]
@LEMONed The thumbnails icons are not working properly. This is what I get in the examples directory:

<img width="672" alt="image" src="https://user-images.githubusercontent.com/223439/51211929-e8de3e80-18e4-11e9-9fbf-3f114e684846.png">

I've used my own compiled version (because I reverted the colors again - sorry classic NFOs are white on black), but I also tried yours.

**Comment by LEMONed** [comment]
> @LEMONed The thumbnail icons are not working properly. This is what I get in the examples directory:
> 
> <img alt="image" width="672" src="https://user-images.githubusercontent.com/223439/51211929-e8de3e80-18e4-11e9-9fbf-3f114e684846.png">
> 
> I've used my own compiled version (because I reverted the colors again - sorry classic NFOs are white on black), but I also tried yours.

I didn't notice this. Looks like `webview` needs to be updated with `wkwebview`. I'll take a closer look when I have time.

**Comment by tessus** [comment]
I tried to use WKWebView a few days ago - with the result of no icons at all. But maybe you have more luck.

**Comment by exekutive** [comment]
> If you find it broken in 10.14.1, here's one rebuilt with Xcode 10.1, tested and working:
> 
> https://www.dropbox.com/s/9fmy5lqnh2kghr9/QuickNFO_Nov2018.zip?dl=0
> 
> and REMEMBER: ❌`~/Library/Quicklook/` ✅`/Library/Quicklook/`

@LEMONed 
File has been deleted. Can you please repost?
